### PR TITLE
I think it looks good for the most part, but I have questions

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

**Comment by Gr8-Dayne** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

**Comment by Gr8-Dayne** [comment]
Unit tests need to be written still.

File: challenges/src/main/java/linkedlist/LinkedList.java [hunk 1]
```diff
@@ -77,6 +77,75 @@ public String toString(){
 
     }
 
+    // Code Challenge 06 (add to end, before, and after)
+
+    public void addToEnd(int value) {
```
Comment (JamesDansie) [inline challenges/src/main/java/linkedlist/LinkedList.java]: try to have your methods match the problem assignment. So instead of addToEnd, use append. It makes it easier for me to find stuff.

Comment (Gr8-Dayne) [reply challenges/src/main/java/linkedlist/LinkedList.java]: I will do so going forward, sorry about that.
